fearless/Common/Protocols/AppUpdatePresentable.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+protocol AppUpdatePresentable {
+    func showAppstoreUpdatePage()
+}
+
+extension AppUpdatePresentable {
+    func showAppstoreUpdatePage() {
+        if let url = URL(string: URLConstants.appstoreLink) {
```

Comment:
if let url = URL(string: URLConstants.appstoreLink), UIApplication.shared.canOpenURL(url)

to avoid crashes if url is broken or unavailable for some reason